packages/inngest/src/components/execution/als.ts

```diff
@@ -0,0 +1,52 @@
+import { type Context } from "../../types.js";
+
+export interface AsyncContext {
+  ctx: Context.Any;
+}
```

Comment:
If found, this is the value returned when a developer calls `getAsyncCtx()` in userland code. It will likely expand to contain more information over time.